# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/compsize/hotfix-include.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

diff -urN compsize-1.5.orig/compsize.c compsize-1.5/compsize.c
--- compsize-1.5.orig/compsize.c	2021-02-08 00:05:36.000000000 +0100
+++ compsize-1.5/compsize.c	2026-07-25 23:14:04.750350143 +0200
@@ -16,6 +17,8 @@
 #include <signal.h>
 #include "radix-tree.h"
 #include "endianness.h"
+#include <errno.h>
+#include <string.h>
 
 #if defined(DEBUG)
     #define DPRINTF(fmt, args...) fprintf(stderr, fmt, ##args)
